src/components/Login.js

```diff
@@ -0,0 +1,147 @@
+import { useState } from "react";
+import * as React from "react";
+import { connect } from "react-redux";
+import auth from "../modules/auth";
+import {
+  View,
+  Text,
+  StyleSheet,
+  TouchableOpacity,
+  TextInput,
+} from "react-native";
+
+const Login = (props) => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+
+  const onSubmithandler = async () => {
+    try {
+      const response = await auth.signIn(email, password);
+      response.success &&
```

Comment:
Sign_in response actually does not contain any success field, so this will not work.
Only the sign_up does I think

Reply:
ok then prove it ;-)